CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Changed
 - More strict types and type hinting
+- `AdapterInterface` does not extend `ConfigurableInterface` anymore
+- `Http` Adapter does not implement `ConfigurableInterface` anymore
```

Comment:
The Http Adapter can handle timeouts:
```
'timeout' => $this->timeout ?? $endpoint->getTimeout(),
```
 So instead of removing the ConfigurableInterface, shouldn't we properly adopt it?

Reply:
No the `Http` adapter does not support any options.

The line you mentioned is like this on master:
```php
'timeout' => $this->timeout,
```

and is just part of the context used for the http request? It has nothing to do with the `ConfigurableInterface`? I don't see any use of any option set on the adapter itself.

Reply:
@mkalkbrenner or do you mean we should make the `timeout` configurable (via `ConfigurableInterface` and an option)? Not sure that is a good idea. We already have `TimeoutAwareInterface::setTimeout` for that.

Reply:
You're right.